lib/src/key/table/nt.rs

```diff
@@ -0,0 +1,158 @@
+//! Stores a LIVE SELECT query notification on the table, ordered by time
+use crate::dbs::node::Timestamp;
+use derive::Key;
+use serde::{Deserialize, Serialize};
+
+/// Nt is used to track live query notifications for remote nodes (nodes, from which
+/// the notification wasn't generated, but which the notification belongs to).
+/// They are ordered by time, where time depends on the storage engine.
+///
+/// Notifications are on a different prefix to the table live queries, because when we scan table
+/// live queries, we do not want to pick up notifications.
+///
+/// The value of the entry is a Notification model.
+#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Serialize, Deserialize, Key)]
+pub struct Nt<'a> {
+	__: u8,
+	_a: u8,
+	pub ns: &'a str,
+	_b: u8,
+	pub db: &'a str,
+	_c: u8,
+	pub tb: &'a str,
+	_d: u8,
+	_e: u8,
+	_f: u8,
+	// We will only be interested in a specific lq notification list
+	#[serde(with = "uuid::serde::compact")]
+	pub lq: uuid::Uuid,
+	_g: u8,
+	// We want the notifications to be ordered by timestamp
+	pub ts: Timestamp,
+	_h: u8,
+	// If timestamps collide, we still want uniqueness
+	#[serde(with = "uuid::serde::compact")]
+	pub nt: uuid::Uuid,
```

Comment:
I was wondering if we should use a u16 (maybe too small) or a u32-based counter rather than a UUID (128bits). For compactness

Reply:
We have tests for compactness so it is stored 128. We use this type because there are issues with serialisation. DON'T TOUCH IT! :P